docs/reference/ml/anomaly-detection/ml-configuring-alerts.asciidoc

```diff
@@ -0,0 +1,85 @@
+[role="xpack"]
+[[ml-configuring-alerts]]
+= Configuring {anomaly-detect} alerts
+
+{anomaly-detect-cap} alerts run scheduled checks on an {anomaly-job} or a group 
+of jobs to detect anomalies with certain conditions. If an {anomaly} meets the 
+conditions, the alert triggers the defined action. For example, you can create 
+an alert that checks an {anomaly-job} in every fifteen minutes for critical 
+anomalies and notify you in an email. This page helps you to configure an 
+{anomaly-detect} alert. To learn more about alerts in the {stack}, refer to 
+{kibana-ref}/alerting-getting-started.html#alerting-getting-started[Alerting and Actions].
+
+
+[[creating-anomaly-alerts]]
+== Creating an alert
+
+{anomaly-detect-cap} alerts can be created in the {anomaly-job} wizard, from the 
+job list or under **Stack management > Alerts and Actions**. On the *Create 
+alert* template, select *{anomaly-detect-cap} alert* under the {ml-cap} section, 
+then give a name to the alert and optionally provide tags.
+
+[role="screenshot"]
+image::images/ml-anomaly-alert-type.jpg["Creating an anomaly detection alert"]
+
+Specify the time interval for the alert to check detected anomalies. It is 
+recommended to select an interval that is similar to the bucket span of the 
+associated job. You can also select a notification option by using the _Notify_ 
+selector. There are three notification options:
+
+* Notify only on status change: actions are triggered when the alert status 
+  changes.
+* Notify every time alert is active: actions are triggered every time when the 
+  alert checks changes.
+* Notify on a custom action interval: actions run by using an interval that you 
+  configured.
+  
+Select the {anomaly-job} or the group of {anomaly-jobs} that is checked by the 
+alert. If you assign additional jobs to the group, the alert automatically 
+checks the new jobs the next time when it runs.
+
+You can select the result type of the {anomaly-job} that triggers the alert. 
+{anomaly-detect-cap} produces three types of results: bucket, record, and 
+influencer. Bucket result type is selected by default.
+
+[role="screenshot"]
+image::images/ml-anomaly-alert-severity.jpg["Selecting result type, severity, and test interval"]
+
+For each alert, you can configure the `anomaly_score` that triggers it. The 
+`anomaly_score` indicates the significance of a given anomaly compared to 
+previous anomalies. The default severity threshold is 75 which means every 
+anomaly with an `anomaly_score` of 75 or higher triggers the alert.
+
+You can also test the configured conditions and check the sample results by 
+providing a valid interval in the pertinent field. The generated preview 
+contains the number of potentially created alert instances during the relative 
+time range you defined.
```

Comment:
> potentially created alert instances...

If you're filling in this data against a job as part of the job creation wizard (i.e. the job has never started yet), is this option still available?  Does that mean you have to wait while it starts the job and generates these scores?

Reply:
I made it clearer in the **Creating an alert** section that you need to start a job to create an alert for it. Even if you don't have any anomalies in the interval you selected, the option is still available and the test shows that the alert was triggered 0 times in the interval.